src/controllers/playback/video/index.js

```diff
@@ -402,6 +402,14 @@ import { appRouter } from '../../../components/appRouter';
             }
         }
 
+        function onFullscreenChanged() {
+            if (!layoutManager.tv) {
+                updateFullscreenIcon();
+            } else if (!playbackManager.isFullscreen(currentPlayer)) {
+                appRouter.back();
+            }
```

Comment:
This breaks playback on webOS 1.2 (emulator) because there is no `webkitIsFullScreen` property: https://github.com/jellyfin/jellyfin-web/blob/f5ea7bc83081d7f3b5a38b00cbbe3b19a101316f/src/components/playback/playbackmanager.js#L1452